templates/desktop/flavours.html

```diff
@@ -14,17 +14,88 @@ <h1>Ubuntu flavours</h1>
       <p class="p-heading--3">What are Ubuntu flavours?</p>
       <p>Ubuntu flavours offer a unique way to experience Ubuntu, each with their own choice of default applications and settings. Ubuntu flavours are owned and developed by members of our global community and backed by the full Ubuntu archive for packages and updates.</p>
     </div>
-    <div class="col-6 u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/e10bc9c7-ubuntu-flavors-cloud.png",
-        alt="",
-        width="300",
-        height="276",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
+      <div class="col-6 p-logo-section">
```

Comment:
We tend to hide non-essential images on smaller breakpoints, could you please add the following classes:

```suggestion
      <div class="col-6 p-logo-section u-hide--small u-hide--medium">
```